visualizer/index.html

```diff
@@ -165,6 +166,50 @@
       color: white;
     }
 
+    #actions {
+      display: flex;
+      flex: 1;
+      position: fixed;
+      right: 20px;
+      margin-top: 40px;
+    }
+    #actions .actionColumn {
+      margin-left: 15px;
+    }
+    #actions .actionEntries {
+      flex-direction: column;
```

Comment:
You don't need this. `flex-direction` is a property for _flex containers_ (i.e., elements that have `display: flex`), not _flex items_. Here's a good guide to flexbox if you haven't seen it: https://css-tricks.com/snippets/css/a-guide-to-flexbox/